spacy/lookups.py

```diff
@@ -120,7 +120,7 @@ def from_bytes(self, bytes_data, **kwargs):
         """
         self._tables = OrderedDict()
         for key, value in srsly.msgpack_loads(bytes_data).items():
-            self._tables[key] = Table(key)
+            self._tables[key] = Table(key, value)
             self._tables[key].update(value)
```

Comment:
This line can be removed because the data is added in the line above.

Reply:
Thanks, I've removed the line.